UnitsNet/Scripts/UnitDefinitions/BrakeSpecificFuelConsumption.json

```diff
@@ -0,0 +1,43 @@
+﻿{
+  "Name": "BrakeSpecificFuelConsumption",
+  "BaseUnit": "KilogramPerJoule",
+  "XmlDoc": "Brake specific fuel consumption (BSFC) is a measure of the fuel efficiency of any prime mover that burns fuel and produces rotational, or shaft, power. It is typically used for comparing the efficiency of internal combustion engines with a shaft output.",
+  "Units": [
+    {
+      "SingularName": "GramPerKiloWattHour",
+      "PluralName": "GramsPerKiloWattHour",
+      "FromUnitToBaseFunc": "x/3.6e9",
```

Comment:
Is this right? 
I found `1 gram/kilowatt-hour = 2.777777777778 x 10-10 kilogram/joule` on this [calculator](http://www.translatorscafe.com/cafe/units-converter/fuel-efficiency-mass/gram-per-kilowatt/hour-5bg/kw*h5d-to-pound-per-horsepower/hour/)